Content.Shared/_DV/Clothing/Systems/SharedShockOnUnequipSystem.cs

```diff
@@ -0,0 +1,18 @@
+using Content.Shared._DV.Clothing.Components;
+using Content.Shared.Examine;
+
+namespace Content.Shared._DV.Clothing.Systems;
+
+public abstract class SharedShockOnUnequipSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<ShockOnUnequipComponent, ExaminedEvent>(OnExamined);
+    }
+    private void OnExamined(Entity<ShockOnUnequipComponent> selfUnremovableClothing, ref ExaminedEvent args)
```

Comment:
Blank line between functions
```suggestion
    }

    private void OnExamined(Entity<ShockOnUnequipComponent> selfUnremovableClothing, ref ExaminedEvent args)
```